connect/utils.go

```diff
@@ -0,0 +1,15 @@
+package connect
+
+import "fmt"
+
+func logGreen(text string) string {
```

Comment:
not sure about the "log" prefix, as technically it does not log, but just converts the text color. How about toGreen(), toRed() and toBold() ?

Reply:
I wanted to keep the naming from original code but these are not very important. I renamed to `greenText()` and `redText()`.